XamarinCommunityToolkit/Views/SegmentedView/iOS/SegmentedViewRenderer.ios.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using System.Linq;
+using Xamarin.CommunityToolkit.iOS.UI.Views;
+using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms;
+using UIKit;
+using Xamarin.Forms.Platform.iOS;
+using System.Threading.Tasks;
+using System.Threading;
+using Foundation;
+using Xamarin.Forms.Internals;
+using Xamarin.CommunityToolkit.Extensions.iOS;
+
+[assembly: ExportRenderer(typeof(SegmentedView), typeof(SegmentedViewRenderer))]
+
+namespace Xamarin.CommunityToolkit.iOS.UI.Views
+{
+	[Xamarin.Forms.Internals.Preserve(AllMembers = true)]
```

Comment:
Can you remove this? We want to make this lib linker safe.

Reply:
Done.